### PR TITLE
test(distributed): add RCCL autoport concurrent smoke test

### PR DESCRIPTION
**Pull request by rebel-jongwonlee**
- Add test_tp_pp_autoport.py to exercise the autoport / unique-id init path by running two parallel subprocesses of test_tp_pp.py with RCCL_PORT_GEN=1, disjoint RBLN_DEVICES (0-3 and 4-7), and distinct MASTER_PORT values.
 - Skip the test when fewer than eight RBLN devices are available so runs on smaller hosts are reported as skipped instead of failing. Marked with pytest.ci and single_worker.i

## Summary of Changes

> *What does this PR do? What feature, fix, or improvement does it bring?*

---

## Related Issues / Tickets

* Resolves #
* Related to #

---

## Type of Change

- [ ] `feature` — New capability or functionality
- [ ] `core` — Changes to RBLN backend, device layer, C++ extension, op registration, or `torch.compile` integration
- [ ] `fix` — Bug fix
- [ ] `perf` — Performance improvement
- [ ] `refactor` — Code improvement without behavior change
- [ ] `docs` — Documentation only
- [ ] `other` — Other (describe below)

---

## Affected Modules

- [ ] Backend
- [ ] Device
- [ ] Ops/Kernels
- [ ] `torch.compile`
- [ ] C++ extension (`torch_rbln/csrc`)
- [ ] Memory
- [ ] Build/Tools
- [ ] Docs

---

## How to Test (if applicable)

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

## Screenshots / Logs (if applicable)

---

## Checklist

* [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format
* [ ] This PR is linked to a corresponding issue — see [Contributing to torch-rbln](docs/CONTRIBUTING.md)
* [ ] Linting passes — see [Linting](docs/LINTING.md)
* [ ] All CI tests pass — see [CI/CD Workflows](docs/WORKFLOWS.md)
* [ ] The test method is described, and the expected result is clearly stated (if applicable)
* [ ] Relevant documentation has been updated (if applicable)

---

## Notes

---

### CODE REVIEW

**Review by rebel-seungchul — APPROVED** [review]
LGTM

File: torch_rbln/_internal/rdma_env.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+def apply_default_rbln_network_environment() -> None:
+    """Apply control-plane defaults and auto-fill ``RBLN_RDMA_IP`` when unset.
+
+    See module docstring for ``RBLN_LOCAL_IP`` / ``RBLN_ROOT_IP`` and
+    ``RBLN_FORCE_LOOPBACK_CONTROL_PLANE``. Safe to call once before loading
+    ``librbln``. Non-empty ``RBLN_RDMA_IP`` is left unchanged.
+    """
+    _apply_control_plane_ips()
+
+    existing = os.environ.get("RBLN_RDMA_IP", "").strip()
```
Comment (rebel-seungchul) [inline torch_rbln/_internal/rdma_env.py:453]: It would be good to print info log for configured RDMA_IP.

**Review by rebel-clee — APPROVED** [review]
LGTM
